docs/source/device_class.rst

```diff
@@ -0,0 +1,99 @@
+.. _device_class:
+
+The Device class
+================
+
+The pyQuil Device class provides useful information for learning about, and working with, Rigetti's available QPUs. One may query for available devices using the ``get_devices`` function:
+
+.. code:: python
+
+    from pyquil.api import get_devices
+
+    devices = get_devices(as_dict=True)
+    # E.g. {'19Q-Acorn': <Device 19Q-Acorn online>, '8Q-Agave': <Device 8Q-Agave offline>}
+
+    acorn = devices['19Q-Acorn']
+
+
+The ``acorn`` Device object holds useful information regarding the QPU, including
```

Comment:
The variable ``acorn`` points to a ``Device`` object that holds useful information regarding the QPU, including: